releasenotes/notes/neutron-cve-37a7821967a36779.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Updated Neutron container image tags to fix CVE-2024-53916. See `#2037002
+    <https://review.opendev.org/c/openstack/neutron/+/896509>`__ for more
```

Comment:
Did you mean to link to https://bugs.launchpad.net/neutron/+bug/2037002 instead?

Reply:
Yep, spot on